app/helpers/assignment_index_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module AssignmentIndexHelper
+  def cal_day(day:)
```

Comment:
Just to confirm that it being `nil` is okay when `day != Date.current`?

Also I think there was some discussion of whether `Date.current` is what you actually want here.